### DISCUSSION TITLE
Squad / skilltester - tab

### DISCUSSION THREAD

**Discussion by GPR6** [ho-dev/HattrickOrganizer#2103] [Dev]
Request for display order values.
Now is example: passable (6,00)
Request: 6.0 (passable)

Why? 
Numbers are easier to understand than long names. Especially when English is not your first language.
Besides, long names like extra-terrestial is not fully displayed in the window. I see for example:
"extra-terresti..." 
"outstanding (..."
"supernatural (..."
and no number. 

and little graphic improvment.
Add on the status bar (next to the "HO" icon) the name of the software as... "Hattrick Organizer" 
and add the name "Hattrick Organizer" to the software tab on the Windows taskbar - at the moment the tab have only "HO" icon.

Also I dont see in options in menu possibilities on change language.

**Comment by wsbrenk** [comment]
Thanks for these ideas

Concerning Language setting take this way -> File->Settings->Language combo box

**Comment by GPR6** [comment]
Another solution: expanding the table so that it displays all data. But word have another lenght in various languages so meybe is more difficult to introducing / adjustment .

Another thing. In table Lineup / Lineup I prefer to move tab "team captain" and "set pieces taker" on right side.
Maybe this is to get used to but now these option are confusing to me with setting the first 11.
